atm.js

```diff
@@ -1,3 +1,91 @@
 $(document).ready(function(){
-// Uh oh -- it's saying `$` is undefined! Something's missing from `index.html`...
+
 });
+
+checkingDeposit = $("#checking").find('.deposit');
+checkingWithdraw = $("#checking").find('.withdraw');
+savingsDeposit = $("#savings").find('.deposit');
+savingsWithdraw = $('#savings').find('.withdraw');
+checkingBalance = 0;
```

Comment:
really the find methods are unnecessary $(".checking .deposit") is enough; again, the string passed into $ is just a css selector. Using `.find()` is an unnecessary computational expense.